src/backend/aspen/api/utils/repo_metadata.py

```diff
@@ -0,0 +1,86 @@
+from typing import Iterable, Mapping, Set, Tuple
```

Comment:
This is mostly renaming the gisaid.py file and updating the code to support extra columns